lib/tasks/migrate_pacbio_aliquot_data.rake

```diff
@@ -85,6 +85,11 @@ namespace :pacbio_aliquot_data do
       # Skip if aliquots already exist (shouldn't happen but just in case)
       next if pool.aliquots.any?
 
+      # Set defaults if they don't exist
+      pool.volume = pool.volume || 0
+      pool.concentration = pool.concentration || 0
+      pool.template_prep_kit_box_barcode = pool.template_prep_kit_box_barcode || '033000000000000000000'
```

Comment:
If we are using the same default values across libraries and pools, would it make sense to have these in somewhere common to both?